docs/MachineConfiguration.md

```diff
@@ -201,7 +201,14 @@ spec:
 **Note:** The RT kernel lowers throughput (performance) in return for improved worst-case latency bounds. This feature is intended only for use cases that require consistent low latency. For more information, see the [Linux Foundation wiki](https://wiki.linuxfoundation.org/realtime/start) and the [RHEL RT portal](https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux_for_real_time/8/).
 
 ### RHCOS Extensions
-RHCOS is a minimal OCP focused OS which provides capabilities common across all the platforms. With extensions support, OCP 4.6 and onward users can enable a limited set of additional functionality on the RHCOS nodes. In OCP 4.6 the supported extensions is `usbguard`. In OCP 4.8 the supported extensions are `usbguard` and `sandboxed-containers`.  In OCP 4.11 the supported extensions are `usbguard`, `sandboxed-containers`, and `kerberos`. In OCP 4.14 the supported extensions are `usbguard`, `sandboxed-containers`, `kerberos`, `ipsec` and `wasm`.
+RHCOS is a minimal OCP focused OS which provides capabilities common across all the platforms. With extensions support, OCP 4.6 and onward users can enable a limited set of additional functionality on the RHCOS nodes.
+| OCP version   |     Supported extensions     |
+| ------------- | ---------------------------- |
+| 4.6           |  `usbguard`                  |
+| 4.8           |  `usbguard`, `sandboxed-containers`    |
+| 4.11          |  `usbguard`, `sandboxed-containers`, `kerberos`   |
+| 4.14          |  `usbguard`, `sandboxed-containers`, `kerberos`, `ipsec`, `wasm`   |
+| 4.15          |  `usbguard`, `sandboxed-containers`, `kerberos`, `ipsec`, `wasm`, `kernel-64k`   |
```

Comment:
Thanks for the table conversion :).

I've now realized that the rt extension is not listed there, so this one likely should not be there but in the section above with the rt kernel.

Reply:
good catch ! I updated the `### KernelType` section above